src/utils/utils_core.h

```diff
@@ -6,8 +6,5 @@
 #ifndef __UTILS_CORE_H__
 #define __UTILS_CORE_H__
 
```

Comment:
File is now empty, so i believe it can be removed.

Reply:
Done.